pom.xml

```diff
@@ -14,4 +14,12 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
+    <dependencies>
```

Comment:
this will not work because OnlineServices is not in the maven central repository. you will need to specify the path to the locally built jar. make sure to make the path generic in order to also work for other people as if you put the specific path it will also include your windows user, the built jar will need to be placed in the GeneralService root directory